src/util.js

```diff
@@ -1545,8 +1545,12 @@ export const importText = (itemsRanked, inputText) => {
   if (numLines === 1) {
 
     const focusOffset = window.getSelection().focusOffset
-    const strippedText = strip(text)
-    const newValue = destKey.slice(0, focusOffset) + strippedText + destKey.slice(focusOffset)
+    let strippedText = strip(text)
+
+    const newText = (destKey !== '' ? ' ': '') + strippedText
```

Comment:
Since `strippedText` is only used once, and its definition is so small, I would suggest inlining it:

```js
const newText = (destKey ? ' ' : '') + strip(text)
```